SheetMetalBaseCmd.py

```diff
@@ -118,7 +118,7 @@ def __init__(self, obj, sketch):
         obj.addProperty(
             "App::PropertyLength", "thickness", "Parameters", _tip_
         ).thickness = 1.0
-        _tip_ = translate("App::Property", "Relief Type")
```

Comment:
We believe that "Relief Type" is wrong, it's correct?

Reply:
yes. I believe It should be Wall Thickness. Shall I fix it?
Also, does the script understand the 'translate' command, or shall I replace if with the longer 'FreeCAD.Qt.Translate'?

Reply:
I made the change, `translate()` is ok

Reply:
sorry if you missed my edit, it should be  'Bend Plane' not wall thickness...

Reply:
ok